proto/mmr.proto

```diff
@@ -3,7 +3,11 @@ package mmr;
 
 import "digest.proto";
 
+// Represents an MMR delta.
 message MmrDelta {
+    // The number of trees in the forest (latest block number + 1).
     uint64 forest = 1;
```

Comment:
I think this should be: "The number of leaf nodes in the MMR". It also does contain information about the number of trees. This can be computed by counting the number of 1's in the `forest` value.